package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@justeat/fozzie",
   "title": "Fozzie – Just Eat UI Web Framework",
   "description": "UI Web Framework for the Just Eat Global Platform",
-  "version": "0.80.0",
+  "version": "0.83.0",
```

Comment:
Should this jump to 3 versions?

Reply:
Ah actually you have two other PRs before, ignore :)

Reply:
i have pr's ope for 81 and 82 - will be merged / rebased in that order 👍